src/Entity/IdeasWorkshop/Thread.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityIdentityTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Thread
+{
+    use EntityIdentityTrait;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Answer", inversedBy="threads")
+     */
+    private $answer;
+
+    /**
+     * @ORM\OneToMany(targetEntity="Comment", mappedBy="thread")
+     */
+    private $comments;
+
+    public function __construct()
+    {
+        $this->comments = new ArrayCollection();
+    }
+
+    public function getAnswer(): Answer
+    {
+        return $this->answer;
+    }
+
+    public function setAnswer($answer): void
+    {
+        $this->answer = $answer;
+    }
+
+    public function addComment(Comment $comment): void
+    {
+        $this->comments->add($comment);
```

Comment:
missing `if (!$this->comments->contains($comment))`

Reply:
:heavy_check_mark: